### PR TITLE
(PLATFORM-2815) Support wikia-dev.us/pl in impactFooter

### PR DESCRIPTION
**Pull request by Grunny**
/cc @Wikia/core-platform-team

### CODE REVIEW

File: extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php [hunk 1]
```diff
@@ -242,6 +242,7 @@ function wfDevBoxResourceLoaderGetConfigVars( &$vars ) {
 	global $wgDevelEnvironment;
 
 	$vars['wgDevelEnvironment'] = $wgDevelEnvironment;
+	$vars['wgWikiaDatacenter'] = $wgWikiaDatacenter;
```
Comment (WikiaTech) [inline extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php]: MAJOR Remove this unused "$wgWikiaDatacenter" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 27 issues
* MAJOR 9 major
* MINOR 5 minor
* INFO 13 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [Special_DevBoxPanel.php#L23](https://github.com/Wikia/app/blob/c37b12fcf78a2691a9fafd9df712de6682bc03e7/extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php#L23): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. MAJOR [Special_DevBoxPanel.php#L26](https://github.com/Wikia/app/blob/c37b12fcf78a2691a9fafd9df712de6682bc03e7/extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php#L26): Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
1. MAJOR [Special_DevBoxPanel.php#L70](https://github.com/Wikia/app/blob/c37b12fcf78a2691a9fafd9df712de6682bc03e7/extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php#L70): Take the required action to fix the issue indicated by this "FIXME" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1134)
1. MAJOR [Special_DevBoxPanel.php#L332](https://github.com/Wikia/app/blob/c37b12fcf78a2691a9fafd9df712de6682bc03e7/extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php#L332): Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
1. MAJOR [Special_DevBoxPanel.php#L349](https://github.com/Wikia/app/blob/c37b12fcf78a2691a9fafd9df712de6682bc03e7/extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php#L349): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. MAJOR [Special_DevBoxPanel.php#L351](https://github.com/Wikia/app/blob/c37b12fcf78a2691a9fafd9df712de6682bc03e7/extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php#L351): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. MAJOR [Special_DevBoxPanel.php#L358](https://github.com/Wikia/app/blob/c37b12fcf78a2691a9fafd9df712de6682bc03e7/extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php#L358): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. MAJOR [Special_DevBoxPanel.php#L360](https://github.com/Wikia/app/blob/c37b12fcf78a2691a9fafd9df712de6682bc03e7/extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php#L360): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. MINOR [Special_DevBoxPanel.php#L23](https://github.com/Wikia/app/blob/c37b12fcf78a2691a9fafd9df712de6682bc03e7/extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php#L23): 2 statements were found on this line. Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)
1. MINOR [Special_DevBoxPanel.php#L28](https://github.com/Wikia/app/blob/c37b12fcf78a2691a9fafd9df712de6682bc03e7/extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php#L28): Define a constant instead of duplicating this literal "DevBoxPanel" 6 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 26 issues
* MAJOR 8 major
* MINOR 5 minor
* INFO 13 info

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [Special_DevBoxPanel.php#L23](https://github.com/Wikia/app/blob/2287af0995fd4b54ed3ca971248b59c676f92ad5/extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php#L23): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. MAJOR [Special_DevBoxPanel.php#L26](https://github.com/Wikia/app/blob/2287af0995fd4b54ed3ca971248b59c676f92ad5/extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php#L26): Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
1. MAJOR [Special_DevBoxPanel.php#L70](https://github.com/Wikia/app/blob/2287af0995fd4b54ed3ca971248b59c676f92ad5/extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php#L70): Take the required action to fix the issue indicated by this "FIXME" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1134)
1. MAJOR [Special_DevBoxPanel.php#L332](https://github.com/Wikia/app/blob/2287af0995fd4b54ed3ca971248b59c676f92ad5/extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php#L332): Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
1. MAJOR [Special_DevBoxPanel.php#L349](https://github.com/Wikia/app/blob/2287af0995fd4b54ed3ca971248b59c676f92ad5/extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php#L349): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. MAJOR [Special_DevBoxPanel.php#L351](https://github.com/Wikia/app/blob/2287af0995fd4b54ed3ca971248b59c676f92ad5/extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php#L351): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. MAJOR [Special_DevBoxPanel.php#L358](https://github.com/Wikia/app/blob/2287af0995fd4b54ed3ca971248b59c676f92ad5/extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php#L358): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. MAJOR [Special_DevBoxPanel.php#L360](https://github.com/Wikia/app/blob/2287af0995fd4b54ed3ca971248b59c676f92ad5/extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php#L360): Add curly braces around the nested statement(s). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
1. MINOR [Special_DevBoxPanel.php#L23](https://github.com/Wikia/app/blob/2287af0995fd4b54ed3ca971248b59c676f92ad5/extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php#L23): 2 statements were found on this line. Reformat the code to have only one statement per line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)
1. MINOR [Special_DevBoxPanel.php#L28](https://github.com/Wikia/app/blob/2287af0995fd4b54ed3ca971248b59c676f92ad5/extensions/wikia/Development/SpecialDevBoxPanel/Special_DevBoxPanel.php#L28): Define a constant instead of duplicating this literal "DevBoxPanel" 6 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)

**Comment by jcellary** [comment]
LGTM - I guess it would be good to define these domains globally instead of copying them in code around the repo, but I see this part is in javascript, so I don't know if that's feasible.

**Review by themech — APPROVED** [review]

**Comment by ludwikkazmierczak** [comment]
Tested and Accepted

**Pull request merged**